packages/extension-inject/src/types.ts

```diff
@@ -3,6 +3,7 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { Signer as InjectedSigner } from '@polkadot/api/types';
+import PostMessageProvider from '@polkadot/extension/page/PostMessageProvider';
```

Comment:
Circular dep

Reply:
Can just create an interface, i.e. `InjectedProvider`, the provider can implement it. (Slight extra work, but probably the more correct approach)